charts/dapr/charts/dapr_rbac/templates/ClusterRoleBinding.yaml

```diff
@@ -8,9 +8,47 @@ subjects:
   namespace: {{ .Release.Namespace }}
 roleRef:
   kind: ClusterRole
-  name: cluster-admin
+  name: dapr-operator-admin
   apiGroup: rbac.authorization.k8s.io
 ---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: role-tokenreview-binding
+subjects:
+- kind: ServiceAccount
+  name: dapr-operator
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:auth-delegator
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: dapr-operator-admin
+  namespace: default
+rules:
+- apiGroups: ["", "dapr.io"]
```

Comment:
apiGroup "" should become "*" in all occurrences.

Reply:
so any (wildcard) api group in all cases?

Reply:
Actually scratch that for now, I think this should be ok. lets test.

Reply:
sorry, I already pushed the change after I saw the tests fail but before I saw this comment. Revert or re-test?